packages/edit-site/src/components/global-styles/preview.js

```diff
@@ -91,7 +91,7 @@ const StylesPreview = ( { label, isFocused } ) => {
 			{ containerResizeListener }
 			<motion.div
 				style={ {
-					height: '100%',
+					height: 150 * ratio,
```

Comment:
This matches the height set on the iframe on line 84. Reviewers might need to expand the file to see it.